test/common/common.go

```diff
@@ -197,11 +241,19 @@ func PatchPlacementRule(namespace, name, targetCluster, kubeconfigHub string) er
 // It assumes that the given filename (stripped of an extension) matches the
 // name of the policy, and that the PlacementRule has the same name, with '-plr'
 // appended.
-func DoCreatePolicyTest(hub, managed dynamic.Interface, policyFile string, templateGVRs ...schema.GroupVersionResource) {
+func DoCreatePolicyTest(
+	hub,
+	managed dynamic.Interface,
+	policyFile string,
+	templateGVRs ...schema.GroupVersionResource,
+) {
 	policyName := strings.TrimSuffix(filepath.Base(policyFile), filepath.Ext(policyFile))
 
-	ginkgo.By("Creating " + policyFile)
-	OcHub("apply", "-f", policyFile, "-n", UserNamespace)
+	ginkgo.By("DoCreatePolicyTest creates " + policyFile + "on namespace" + UserNamespace)
```

Comment:
What are these `ginkgo.By` changes for? These seem like better log messages.

Reply:
These are just some better log messages.

Reply:
Could you make this a log message instead?